src/docs/markdown/automatic-https.md

```diff
@@ -220,3 +222,84 @@ Before attempting any ACME transactions, Caddy will test the configured storage
 Caddy can obtain and manage wildcard certificates when it is configured to serve a site with a qualifying wildcard name. A site name qualifies for a wildcard if only its left-most domain label is a wildcard. For example, `*.example.com` qualifies, but these do not: `sub.*.example.com`, `foo*.example.com`, `*bar.example.com`, and `*.*.example.com`.
 
 To get a wildcard from Let's Encrypt, you simply need to enable the [DNS challenge](#dns-challenge) and use a wildcard domain in your config. We recommend using wildcards only when you have so many subdomains that you would encounter CA rate limits trying to obtain certificates for them all.
+
+
+
+## Examples
+
+Caddy implicitly uses the HTTPS port (default 443) for your [site addresses](/docs/conventions#network-addresses) that don't specify a port explicitly (which would disable automatic HTTPS). 
+
+Automatic HTTPS can be configured via the Caddyfile with the [`auto_https` global option](/docs/caddyfile/options), or via [per-server JSON configuration](/docs/json/apps/http/servers/automatic_https/). The Caddyfile option can be set to either of the following:
+- `disable_redirects` which disables the implicit HTTP->HTTPS redirect.
+- `off` which disables automatic HTTPS altogether, including the HTTP->HTTPS redirect and automatic enabling of TLS for sites that meet the requirements.
+
+| auto_https        | HTTP        | HTTPS       |
+|-------------------|-------------|-------------|
+| *default*         | redirects   | connects    |
+| disable_redirects | unreachable | connects    |
+| off               | connects    | unreachable |
+
+`unreachable` is the default without additional configuration when using the related `auto_https` setting.
+`connects` is the implicitly assigned port.
+
+
+### Disabling automatic HTTPS
+
+For local development environments, you can prevent serving via HTTPS by either specifying `http://` or providing a non-HTTPS port to disable per site, or by adding `auto_https off` to the Caddyfile global options which will change Caddy's implicit port to be the HTTP port (default 80).
```

Comment:
This works in _any_ environment, not just local development.

Reply:
I would like to see this section refer to the activation criteria and then show 1:1 how _not_ meeting those criteria disables auto HTTPS.

Reply:
> This works in any environment, not just local development.

Yes, I could probably have re-worded that better. I had seen the quick-starts guide earlier that was more involved in config/requirements IIRC, and explicitly stated that it wasn't for local environment(eg `localhost` site address). I may have made this mistake here due to train of thought with the localhost examples used, and originally having `local_certs` global option set which I apparently didn't need to do for localhost site address and later removed before committing.

> I would like to see this section refer to the activation criteria and then show 1:1 how not meeting those criteria disables auto HTTPS.

Sure I can do that :+1: